### PR TITLE
feat: new spanner quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New Cloud Spanner quickstart. This takes unique metrics from a prometheus exporter not available with the current spanner integration.

## Pre merge checklist

- [ X] Did you check you NRQL syntax? - Does it work?
- [ X] Did you check your dashboard image quality? -  Do they look good?
- [ X] Did you include an InstallPlan and Documentation reference?
- [ X] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ X] Did you attach images of your dashboards to the PR so we can see them working?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202663307643656

### CODE REVIEW

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
...yJZ8Sc6LVG0XmPMtR+a17Rq/WLAYgGRWgBV65hIo7O+mkG+pjTXk1nZ3dj1PAAeaUWWsYzF5gRQ1k1ToeqMZ9GTk8E/tvxWTQbw8ze+HV4DV5u4gEixOXEOBzF3moRo2PC/c9xmJu1sIm9v+YRKrH4JXCR3+c0aMYYV4o9G9yoUcU1IC98G75cJxpYuFYUhWkNjTYjOEL49Y5fzWR71Rvnwzst/WKPU/oYuz6MkrdVY46ZO28iR2eF+2FbZ3m/j/rYRtzOMgi6SKZp5EWUH/yVGjG8H/T0ExWQUfGNmCKiMI2x0tfErJ5uER/Ce0NDv3HWXuUFPmHhiX9IQ/w61oaJvvLPIf7G+yYIuRJhEdkfMQVAYRxzz4tJAJgGIs/H3RQiN/FXvtNAFGJu8bCN+IwfFAuJX4v8CmNfT/2RhDBZRP4+WkLqlDomMmKiiK7FgahqEGQ2ySK6lruiqyI+Eunp4T4Z8X/wABnzsUCImAzgwz329sGJQkaYEKJQwwVkGFIjJoLI3+MDMi3CK/GxkMC1KOFEir3kNikifXcjfRBtwu4Qj1PaiA9Lou8KYuQhqbOhjSiEF7mDZX4Rj1Oq7uYhQpE7RE1yQpqIPImb2cogdzYUEfkPg5iQaVnEzobetTiF+I51fUwOSMuKD39Avmu1Y10s4pQsUhxUA7gSqSBK0YvcwdJBCCkgSmDfzN0kD/s0EKX41SdfAYR9eERpDQfIdGAIYRElIDezkUxa0KCAKP0BvbETIj2FRQRISB2SJ0CEYIhSlF0XcWRWLwtR8tsshq86ECAUIvmcwi19DGVECETpAyzYH9UCSU+BEHGL3MH6ATMiMaKEW+QOlgkGSIy4pHSPPHEJHArx1/M+TgDpFtw4JUGUphTczFaNERwhAeIvFTezkQwXMQgQCYvcwYJ0NtiIU5qP4tBhChpEiFKN2lVoy3xPG1HC2XURR3NIQCxEwNKFp1SwOQYeovRN84E/Gw2WoIRxEafwc4ozgUaMuIjSB31AxoAqaGAgSmsjAUKmv0wLUVrSy2Ycak3TQZSSuAjTRJRAltIiqgMcMx4jLC9K35AV4FCpNTHZuChJnncp0hRw/haIKEnUczVvxtr71/JRhJIgeet+ul7V0uCzJXdmtQqY/nhrMutQnA9eddVVV1111VVXXXVVMvofAlT4ioxrICQAAAAASUVORK5CYII=)"
```
Comment (aswanson-nr) [inline dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json:22]: Hey @jcountsNR, 
I don't think anyone has tried tossing a base64 image in any quickstart dashboards, I just want to confirm that you see this working on your end. If so, this is a great way to manage dashboard images!

File: quickstarts/gcp/google-cloud-spanner-latency/config.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+keywords:
+  - gcp
+  - google cloud platform
+  - database
+  - newrelic-partner
+
+installPlans: 
+  - cloud-spanner
+
+Dashboard: gcp-cloud-spanner
```
Comment (aswanson-nr) [inline quickstarts/gcp/google-cloud-spanner-latency/config.yml:51]:
```suggestion
dashboards: 
  - gcp-cloud-spanner
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Just saw one field that needs to be cleaned up!

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json (hunk 1 above)
Comment (jcountsNR) [reply dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json:22]: Screen Shot 2022-09-23 at 2 27 59 PM
Here is a screenshot, it's been working for me.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**